libpod/container_internal.go

```diff
@@ -795,7 +795,7 @@ func (c *Container) isStopped() (bool, int32, error) {
 		return true, -1, err
 	}
 
-	return !c.ensureState(define.ContainerStateRunning, define.ContainerStatePaused, define.ContainerStateStopping), c.state.ExitCode, nil
+	return c.ensureState(define.ContainerStateExited), c.state.ExitCode, nil
```

Comment:
Need to add ContainerStateConfigured here too, reinit can return containers to that state from Exited.